README.md

```diff
@@ -96,19 +104,15 @@ await VoxeetSDKExt.breakout.initialize();
 // Create a new room and move participants into it
 const room = {
     name: 'Video games',
```

Comment:
id does not have to be unique ? 
we can reuse already used one ?

Reply:
The ID is generated by this extension.